doc/controls/LoadingView.md

```diff
@@ -6,6 +6,7 @@ Property|Type|Description
 -|-|-
 Source|ILoadable|Gets and sets the source `ILoadable` associated with this control.
 LoadingContent|object|Gets or sets the content to be displayed during loading/waiting.
+LoadingContentTemplate|DataTemplate|Gets or sets the template to be used to display the LoadingContent during loading/waiting.
```

Comment:
Would a LoadingContentTemplateSelector be useful?

Reply:
added LoadingContentTemplateSelector property